GOVERNANCE.md

```diff
@@ -0,0 +1,75 @@
+# OpenJS Standards Group
+
+<!-- yet to add -->
+For the current list of Team members, see the project [README.md](./README.md).
+
+## Members
+
+The [openjs-foundation/standards](https://github.com/openjs-foundation/standards) GitHub
+repository is maintained by the Team and additional Members who are
+added on an ongoing basis.
+
+* Invited to all meetings
+* Can participate in [consensus seeking process](#consensus-seeking-process)
+* Counted towards quorum in [Team Meetings](#team-meetings)
+* Participates in voting
+
+## Team Membership
+
+Team Membership is not time-limited. There is no fixed size of the Team.
+
+There is no specific set of requirements or qualifications for Team Membership beyond these rules.
+
+The following groups automatically qualify for membership and can request to be added to the GitHub team:
+
+* OpenJS Foundation CPC Members
+* OpenJS Foundation Project Maintainers
+
+## Team Meetings
+
+The Team meets bi-weekly on Zoom.us. A designated moderator
+approved by the Team runs the meeting. Each meeting should be
+published to YouTube.
+
+Items are added to the Team agenda that are considered contentious or
+are modifications of governance, contribution policy, Team membership,
+or release process.
+
+The intention of the agenda is not to approve or review all patches;
+that should happen continuously on GitHub and be handled by the larger
+group of Collaborators.
+
+Any community member or contributor can ask that something be added to
+the next meeting's agenda by logging a GitHub Issue. Any Collaborator,
+Team member or the moderator can add the item to the agenda by adding
+the ***standards-agenda*** tag to the issue.
+
+Prior to each Team meeting the moderator will share the agenda with
+members of the Team. Team members can add any items they like to the
+agenda at the beginning of each meeting. The moderator and the Team
+cannot veto or remove items.
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+## Consensus Seeking Process
+
+The Team follows a
+[Consensus Seeking](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+decision-making model.
+
+When an agenda item has appeared to reach a consensus, the moderator
+will ask "Does anyone object?" as a final call for dissent from the
+consensus.
+
+If an agenda item cannot reach a consensus, a Team member can call for
+the item to be decided by a vote or to table the issue to the next
+meeting. In both cases the decision must be seconded by a majority of the Team
+or else the discussion will continue. Simple majority wins. Only Active
+Members participate in a vote.
+
+## Pull Requests
+
+Pull requests require approval by at least two members and must remain open for at least 72 hours before being landed. If there are any objections the Pull Request cannot land until they are resolved.
+
+A pull request that is opposed will be discussed in the meeting.
```

Comment:
I think this can be inferred from the above paragraph.... this is a tiny nit though